runtime/sema/type.go

```diff
@@ -3695,6 +3717,15 @@ func (t *CompositeType) IsExternallyReturnable(results map[*Member]bool) bool {
 	return true
 }
 
+func (t *CompositeType) HasComputedMembers() bool {
+	return t.hasComputedMembers
+}
+
+// SetHasComputedMembers is intended to only be called for deserialization.
+func (t *CompositeType) SetHasComputedMembers(val bool) {
+	t.hasComputedMembers = val
+}
```

Comment:
This might be unnecessary for two possible reasons:
1. If computable members are only present in built-in CompositeTypes and we choose to encode built-in types as an enum then there's no need to ever encode hasComputedMembers because it would always be `false` for any encoded CompositeType.
2. Any CompositeType where `hasComputedMembers == true` is considered not-storable. If a type is not storable then does it ever need to be transmitted? If not then there's no need to encode such types.